src/Language/DifferentialDatalog/Attribute.hs

```diff
@@ -127,6 +134,23 @@ tdefCheckSizeAttr d TypeDef{..} =
          [Attribute{..}] -> err d attrPos $ "Invalid 'size' attribute: size must be an integer between 0 and " ++ show (maxBound::Int)
          _                    -> err d tdefPos $ "Multiple 'size' attributes are not allowed"
 
+{- 'custom_serde' attribute: Tells DDlog not to generate `Serialize` and
```

Comment:
it is not useful to have this documentation just in the code, you have to surface it to the users as well.
there should be a section on annotations in the reference document?

Reply:
There is a section in the tutorial. I'll add the new attribute there.